deepl-connector-demo/src_hd/com/axonivy/connector/deepl/demo/UploadInvoice/UploadInvoiceProcess.p.json

```diff
@@ -0,0 +1,163 @@
+      "config" : {
+        "callSignature" : "start",
+        "input" : {
+          "code" : "out.languages = com.axonivy.connector.deepl.LanguageInfo.all();"
+        },
+        "guid" : "186928C0C594B9C3"
+      },
+      "visual" : {
+        "at" : { "x" : 560, "y" : 496 },
+        "labelOffset" : { "x" : 8, "y" : 35 }
+      },
+      "connect" : { "id" : "f4", "to" : "f7" }
+    }, {
+      "id" : "f7",
+      "type" : "HtmlDialogEnd",
+      "visual" : {
+        "at" : { "x" : 816, "y" : 496 }
+      }
+    }, {
+      "id" : "f16",
+      "type" : "Script",
+      "name" : "set file content and name",
+      "config" : {
+        "output" : {
+          "map" : {
+            "out" : "in",
+            "out.showContent" : "false"
+          },
+          "code" : "in.filePath = in.ivyFile.getAbsolutePath();"
+        }
+      },
+      "visual" : {
+        "at" : { "x" : 704, "y" : 584 },
+        "size" : { "width" : 144, "height" : 60 }
+      },
+      "connect" : { "id" : "f1", "to" : "f0" }
+    }, {
+      "id" : "f13",
+      "type" : "HtmlDialogEventStart",
+      "name" : "fileUpload",
+      "config" : {
+        "guid" : "186928C0C59F098D"
+      },
+      "visual" : {
+        "at" : { "x" : 560, "y" : 584 },
+        "labelOffset" : { "x" : 11, "y" : 32 }
+      },
+      "connect" : { "id" : "f17", "to" : "f16" }
+    }, {
+      "id" : "f0",
+      "type" : "SubProcessCall",
+      "name" : "translate",
+      "config" : {
+        "processCall" : "deepl/translate:Document(File,String,File)",
+        "output" : {
+          "map" : {
+            "out" : "in",
+            "out.ivyFile" : "in.ivyFile",
+            "out.translated" : "result.translated"
+          }
+        },
+        "call" : {
+          "params" : [
+            { "name" : "file", "type" : "File" },
+            { "name" : "targetLanguage", "type" : "String" },
+            { "name" : "ivyFile", "type" : "File" }
+          ],
+          "map" : {
+            "param.targetLanguage" : "in.translate.key()",
+            "param.ivyFile" : "in.ivyFile"
+          }
+        }
+      },
+      "visual" : {
+        "at" : { "x" : 904, "y" : 584 },
+        "icon" : "res:/webContent/logo/deepl-logo-blue.png"
+      },
+      "connect" : { "id" : "f3", "to" : "f2" }
+    }, {
+      "id" : "f2",
+      "type" : "HtmlDialogEnd",
+      "visual" : {
+        "at" : { "x" : 1048, "y" : 584 }
+      }
+    }, {
+      "id" : "f15",
+      "type" : "HtmlDialogEventStart",
+      "name" : "fileUploadTemplate",
+      "config" : {
+        "guid" : "18722511F883455A"
+      },
+      "visual" : {
+        "at" : { "x" : 560, "y" : 664 },
+        "labelOffset" : { "x" : 11, "y" : 32 }
+      },
+      "connect" : { "id" : "f21", "to" : "f19" }
+    }, {
+      "id" : "f18",
+      "type" : "SubProcessCall",
+      "name" : "translate",
+      "config" : {
+        "processCall" : "deepl/translate:DocumentSample(File,String)",
```

Comment:
I'd not create a special start that handles samples ... the demo should send the 'sample.pdf' to the same sub-process start ... as the manually uploaded ones